_posts/2019-05-10-continuous-deployment-the-first-step-on-the-road-to-recovery.markdown

```diff
@@ -0,0 +1,63 @@
+# Continuous Deployment - the first step on your road to recovery
+
+** HMM, does this still fit?**
+[Systems thinking]() in DevOps is about improving the flow of work into production.  Hand-offs, re-work and batching are all enemies of flow.  Teams don't intentionally look to reduce flow.  Hand-offs and approvals are mechanisms introduced in an attempt to tackle the risks of delivery.  Value is generated through change.  Change is the enemy of stability.  XXX said "Systems architecture is a reflection of organisational structure".  ***I say the deployment pipeline is a reflection of the reporting structure.***.  Continuous deployment is a goal for most teams; a lofty ambition that other people achieve.  For Mechanical Rock, continuous deployment is our starting point.  And continuous deployment starts with continuous delivery and a suitable branching strategy.
```

Comment:
You need to re-work this because you are introducing a lot of ideas at once and it's a bit awkward to read.

Reply:
Split the paragraphs.  But I'm assuming a certain level of understanding from the reader here.  If they aren't familiar with the concepts then they need to read about them first.